framework/Metrics/metrics/MetricUtilities.py

```diff
@@ -21,6 +21,7 @@
 import numpy as np
 import math
 import scipy
+from scipy import stats
```

Comment:
why do you need this import? I do not see the place that you are using this function.

Reply:
import removed